gallifreyan.js

```diff
@@ -255,6 +274,8 @@ function doClick(e) {
         deleteLineMode = false;
         redraw();
     }
+    if (addLineMode)
+        addLineMode = false;	//first end has already been placed, now it's a normal selectedLine
```

Comment:
I'd also change the comment here: `don't move both ends anymore; now it's a normal selectedLine with one attached end`